core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -147,8 +148,9 @@ fun LpOutlinedTextFieldMail(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun LpOutlinedTextFieldNumber(
+fun LpOutlinedTextFieldInput(
     modifier: Modifier,
+    enabled:Boolean,
```

Comment:
Oh, please add a default value as it is in one of the description of your ticket:
`To have “enabled” option as parameter passed to function (“true” as the default value), so the user will be unable to edit/modify the text`